src/CommunityToolkit.Maui/Alerts/Toast/Toast.shared.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Alerts;
+
+/// <inheritdoc/>
+public partial class Toast : IToast
+{
+	/// <inheritdoc/>
+	public string Text { get; set; } = string.Empty;
+
+	/// <inheritdoc/>
+	public ToastDuration Duration { get; set; }
+
+	/// <summary>
+	/// Create new Toast
+	/// </summary>
+	/// <param name="message">Toast message</param>
+	/// <param name="duration">Toast duration</param>
+	/// <returns>New instance of Toast</returns>
+	public static IToast Make(
+		string message,
+		ToastDuration duration = ToastDuration.Short)
+	{
+		return new Toast()
```

Comment:
Sine we have this method to create the Toast. We should implement a private or internal constructor to avoid devs to create a new instance without using this method.

Reply:
You're not wrong. 

We're basically matching how Android instantiates `Toast`, via `Toast.Make`, while also csharp-ifying it by allowing devs to initialize it via `new Toast()`.

The hope is that this lowers the friction for devs, allowing them to instantiate `Toast` via the way they're most comfortable/familiar, without needing to look up the docs.  

We did the same implementation for `Snackbar`.

Reply:
I can't see any value-creating an empty Toast. So here I would say to just keep the `Make` method, to create a Toast or change it to be a constructor with two parameters.
IMHO having two ways to create a Toast will be more error-prone than keeping just one way

Reply:
Android Toast has a constructor, but it receives Context as a first parameter.
As for me I always use Toast.Make, but maybe we should keep 2 variants to look similar to the snackbar

Reply:
We don't need to keep it just because Google keeps it. 
We can't change the text of a Toast after calling the `Show` method, so this should be `immutable`. Making this immutable will help us to avoid `bugs` in the future. So can't see any good reason to not restrict the `Toast` creation and make the properties `read-only`.